crates/oxc_cli/src/lib.rs

```diff
@@ -55,27 +54,42 @@ impl Cli {
             })
             .collect::<Vec<_>>();
 
-        let number_of_diagnostics = paths
-            .par_iter()
-            .map(|path| {
-                let diagnostics = Self::lint_path(path);
-                diagnostics
-                    .iter()
-                    .filter(|d| match d.severity() {
-                        // The --quiet flag follows ESLint's --quiet behavior as documented here: https://eslint.org/docs/latest/use/command-line-interface#--quiet
-                        // Note that it does not disable ALL diagnostics, only Warning diagnostics
-                        Some(Severity::Warning) => !self.cli_options.quiet,
-                        _ => true,
-                    })
-                    .for_each(|diagnostic| {
-                        println!("{diagnostic:?}");
-                    });
```

Comment:
Can you revert this part so I can merge first? We'll go through the implementation in discord.

Reply:
Reverted.